eox_core/tests/integration/test_enrollment_integration.py

```diff
@@ -0,0 +1,606 @@
+                                     self.data['endpoint'])
+        response = requests.put(request_url,
+                                data=data,
+                                headers=headers)
+        self.assertEqual(response.status_code, 202)
+
+    def test_update_valid_enrollment_change_is_active_force_post(self):
+        # pylint: disable=invalid-name
+        """
+        Update an existing enrollment using POST with force=True;
+        change is_active flag
+        """
+        create_enrollment(self.data)
+        site1_data = self.data['site1_data']
+        data = {
+            'email': site1_data['user_email'],
+            'course_id': site1_data['course']['id'],
+            'is_active': False,
+            'mode': site1_data['course']["mode"],
+            'force': True,
+        }
+        headers = {
+            'Authorization': 'Bearer {}'.format(site1_data['token']),
+            'Host': site1_data['host'],
+        }
+        expected_response = {
+            'username': site1_data['user_id'],
+            'is_active': False,
+            'course_id': data['course_id'],
+        }
+        request_url = '{}/{}'.format(site1_data['base_url'],
+                                     self.data['endpoint'])
+        response = requests.post(request_url,
+                                 data=data,
+                                 headers=headers)
+        self.assertEqual(response.status_code, 200)
+        response_content = response.json()
+        self.assertDictContainsSubset(expected_response, response_content)
+
+    # NOTE: Mode changes are not working correctly on devstack
+    def test_update_valid_enrollment_change_valid_mode_force_post(self):
+        # pylint: disable=invalid-name
+        """
+        Update an existing enrollment; change mode
+        """
+        create_enrollment(self.data)
+        site1_data = self.data['site1_data']
+        data = {
+            'email': site1_data['user_email'],
+            'course_id': site1_data['course']['id'],
+            'is_active': True,
+            'mode': 'honor',
+            'force': True
+        }
+        headers = {
+            'Authorization': 'Bearer {}'.format(site1_data['token']),
+            'Host': site1_data['host'],
+        }
+        expected_response = {
+            'user': site1_data['user_id'],
+            'is_active': True,
+            'course_id': data['course_id'],
+            'mode': 'honor',
+        }
+        request_url = '{}/{}'.format(site1_data['base_url'],
+                                     self.data['endpoint'])
+        response = requests.put(request_url,
+                                data=data,
+                                headers=headers)
+        self.assertEqual(response.status_code, 200)
+        response_content = response.json()
+        self.assertDictContainsSubset(expected_response, response_content)
+
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
I think maybe you should define this class method before all the tests, right after the definition of setUpClass

Reply:
Done.